docs/src/pages/landing/Users.js

```diff
@@ -83,44 +82,47 @@ const users = [
   },
 ];
 
-const styles = theme => ({
-  root: {
-    padding: theme.spacing(2),
-    minHeight: 160,
-    paddingTop: theme.spacing(5),
-  },
-  container: {
-    marginBottom: theme.spacing(4),
-  },
-  users: {
-    padding: theme.spacing(10, 4, 0),
-  },
-  grid: {
-    marginTop: theme.spacing(5),
-    marginBottom: theme.spacing(5),
-  },
-  img: {
-    margin: theme.spacing(1.5, 3),
-  },
-  amazon: {
-    margin: theme.spacing(2.4, 3, 1.5),
-  },
-  coursera: {
-    margin: theme.spacing(2.3, 3, 1.5),
-  },
-  unity: {
-    margin: theme.spacing(0.5, 3, 1.5),
-  },
-  walmart: {
-    margin: '13px 4px 12px',
-  },
-  button: {
-    margin: theme.spacing(2, 0, 0),
-  },
-});
+const useStyles = makeStyles(
+  theme => ({
+    root: {
+      padding: theme.spacing(2),
+      minHeight: 160,
+      paddingTop: theme.spacing(5),
+    },
+    container: {
+      marginBottom: theme.spacing(4),
+    },
+    users: {
+      padding: theme.spacing(10, 4, 0),
+    },
+    grid: {
+      marginTop: theme.spacing(5),
+      marginBottom: theme.spacing(5),
+    },
+    img: {
+      margin: theme.spacing(1.5, 3),
+    },
+    amazon: {
+      margin: theme.spacing(2.4, 3, 1.5),
+    },
+    coursera: {
+      margin: theme.spacing(2.3, 3, 1.5),
+    },
+    unity: {
+      margin: theme.spacing(0.5, 3, 1.5),
+    },
+    walmart: {
+      margin: '13px 4px 12px',
+    },
+    button: {
+      margin: theme.spacing(2, 0, 0),
+    },
+  }),
+  { name: 'HomeUsers' },
```

Comment:
```suggestion
```

Reply:
Just users will do. But these name are deliberate since it will use useless names otherwise.

Reply:
Yeah, this part of the makeStyles solution isn't great. Maybe we should rename it `LandingUsers`?

Reply:
Why add a prefix that is implicit?

Reply:
Just Users in this case?

Reply:
> So the DevTools -> Source experience can be easier?

Yep. Helps quickly identify what component owns a certain DOM element. Not always but in most cases.

> Maybe just Users in this case?

Yeah I forgot it when moving the files.

Reply:
done